### PR TITLE
added application key

### PR DESCRIPTION
**Pull request by ukcai696969**

### CODE REVIEW

File: versioned_docs/version-0.8/Guides/backups.md [hunk 1]
```diff
@@ -70,3 +70,14 @@ Run `crontab -e` and add the following line:
 
 This will create a backup at /var/www/controlpanel/backups-$(date +\%F).sql every day at midnight.
 Every file will have the date when it was made in the filename, so you can also resolve issues that happened a few days ago.
+
```
Comment (BolverBlitz) [inline versioned_docs/version-0.8/Guides/backups.md:73]: Please follow the structure of the file like this:

# Application Key
<Smal explenation what it is and why its important to keep save>
### Appliaction key backup
<Text>
###Apllication key restore
<Text>

File: versioned_docs/version-0.9/Guides/backups.md [hunk 2]
```diff
@@ -71,3 +71,14 @@ Run `crontab -e` and add the following line:
 
 This will create a backup at /var/www/controlpanel/backups-$(date +\%F).sql every day at midnight.
 Every file will have the date when it was made in the filename, so you can also resolve issues that happened a few days ago.
+
```
Comment (BolverBlitz) [inline versioned_docs/version-0.9/Guides/backups.md:74]: Please follow the structure of the file like this:

```
# Application Key
<Smal explenation what it is and why its important to keep save>
### Appliaction key backup
<Text>
###Apllication key restore
<Text>
```

**Comment by BolverBlitz** [comment]
You also might wanna have a look at `grep ` and `sed` some basic linux programs to format output.
To get just teh key, you could use something like this:
```sh
cat .env | grep "\<APP_KEY\>" | sed 's/APP_KEY=//g'
```

**Comment by GeckoBoy84** [comment]
Even in Bolvers commit, application is spelled wrong!
(Appliaction key)

**Comment by BolverBlitz** [comment]
Thanks @GeckoBoy84 then just fix it ^^

**Pull request merged**